Makefile

```diff
@@ -157,7 +157,12 @@ SWEEP ?= global
 
 sweep:
 	@echo 'WARNING: This will destroy infrastructure. Use only in development accounts.'
-	$(GO) test ./internal/sweep -v -tags=sweep -sweep=$(SWEEP) $(SWEEP_ARGS) -timeout 15m
+	$(GO) test ./internal/sweep -v -sweep=$(SWEEP) $(SWEEP_ARGS) -timeout 15m
+
+SWEEP_CHECK ?= TestCheckSweepers
```

Comment:
Not sure if this should be configurable. I mean, there is only one function. One won't ever run this with different test name. Or am I missing something?